cfme/infrastructure/virtual_machines.py

```diff
@@ -350,44 +350,58 @@ def _nav_to_snapshot_mgmt(self):
                 self.vm.load_details()
                 sel.click(InfoBlock.element("Properties", "Snapshots"))
 
-        def does_snapshot_exist(self):
+        @property
+        def exists(self):
```

Comment:
Great! :+1: I'm happy you changed this as well, based on the previous comments :) This was out of scope, but I'm glad you noticed the issue.